pyatv/mrp/__init__.py

```diff
@@ -48,6 +53,7 @@ def __init__(self, loop, protocol):
         if lookup:
             await self.protocol.send(
                 messages.send_hid_event(lookup[0], lookup[1], True))
+            time.sleep(lookup[2])
```

Comment:
Two things here:
* Why is this needed?
* You must use the asyncio version (asyncio.sleep), this will block the event loop

Reply:
Good catch. I just committed a change to implement asynchronous sleep.

The messages.send_hid_event(arg, arg, True) sends the keydown event to Apple TV, messages.send_hid_event(arg, arg, False) sends keyup. For normal use you want these events to execute back to back with no delay, but a sleep allows for sending "press and hold" button events. Specifically implemented here with the home_hold command.

Reply:
Ah yes, that is correct. I admit that I did not read my own code good enough here... But it looks good to me!